bootstrap/terraform/main.tf

```diff
@@ -81,24 +127,72 @@ module "eks_blueprints" {
 }
 
 #---------------------------------------------------------------
-# EKS Blueprints Addons
+# EKS Addons
 #---------------------------------------------------------------
 
 module "eks_blueprints_kubernetes_addons" {
-  source = "github.com/aws-ia/terraform-aws-eks-blueprints//modules/kubernetes-addons?ref=v4.22.0"
+  source = "github.com/aws-ia/terraform-aws-eks-blueprints-addons"
+
+  cluster_name          = module.eks.cluster_name
+  cluster_endpoint      = module.eks.cluster_endpoint
+  cluster_version       = module.eks.cluster_version
+  oidc_provider         = module.eks.oidc_provider
+  oidc_provider_arn     = module.eks.oidc_provider_arn
+  enable_karpenter      = true  
+  enable_metrics_server = true 
+  enable_prometheus     = true
 
-  eks_cluster_id = module.eks_blueprints.eks_cluster_id
+  depends_on = [module.eks.managed_node_groups]
+}  
 
+module "eks_blueprints_crossplane_addons" {
+  source = "github.com/aws-ia/terraform-aws-eks-blueprints//modules/kubernetes-addons?ref=v4.26.0"
+
+  eks_cluster_id = module.eks.cluster_name
   # Deploy Crossplane
   # Default helm chart and providers values set at https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/modules/kubernetes-addons/crossplane/locals.tf
   enable_crossplane = true
-
+  crossplane_helm_config = {
+    version = "1.11.2"
+    values = [yamlencode({
+      args    = ["--enable-environment-configs","--enable-composition-functions"]
```

Comment:
Can we not enable composition functions for now? A few reasons:
1. We don't have any examples of using this functionality yet. 
2. This feature is still a bit unstable. We can enable it when it becomes more stable. Env config seems to be more stable so I am ok with enabling it. 
3. This does have some security implications since you are allowing it to run arbitrary images in the side car.

Reply:
I agree folks are taking this bootstrap as starting point while not paying attention that Functions come with some caveats

Reply:
OK, I will remove.